xml/en/community.xml

```diff
@@ -5,18 +5,89 @@
 
 <!DOCTYPE article SYSTEM "../../dtd/article.dtd">
 
-<article name="nginx: support"
-         link="/en/support.html"
+<article name="nginx: community"
+         link="/en/community.html"
          lang="en"
-         rev="10">
+         rev="11">
 
+<section name="Community and Governance">
 
-<section name="Commercial Support">
+<para>
+Open source powers much of the technology world.
+NGINX has been a strong part of this movement from its inception.
+As the internet has grown, so have we, from our initial web server to the
+myriad of projects involving JavaScript, Kubernetes, API gateways,
+OpenTelemetry, and WebAssembly.
+NGINX is committed to open source both in ethos and practicality, aiming to
+deliver reliable and cutting-edge technology within a healthy open source
+culture.
+Whether NGINX is part of your open source past or a step in your present,
+we welcome you into the future together.
+</para>
+
+<para>
+To learn more about our commitment, please visit
+the <link url="/en/docs/governance.html">Governance</link>
```

Comment:
bad link, should be:
``<link doc="docs/governance.xml">Governance</link>``

Reply:
Gotcha, so link directly to the underlying XML docs when appropriate.